ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/ApplyMigrationCommand.java

```diff
@@ -15,6 +15,7 @@
 
 package io.confluent.ksql.tools.migrations.commands;
 
+import static io.confluent.ksql.tools.migrations.util.CommandParser.preserveCase;
```

Comment:
This import is unused, right?

Reply:
It's used to preserve case for field names in insert statements. This is being done here instead of in CommandParser, because the client is used to get field names.